pkgs/development/compilers/gcc/common/builder.nix

```diff
@@ -217,19 +217,21 @@ originalAttrs: (stdenv.mkDerivation (finalAttrs: originalAttrs // {
     # Move runtime libraries to lib output.
     moveToOutput "''${targetConfig+$targetConfig/}lib/lib*.so*" "''${!outputLib}"
     moveToOutput "''${targetConfig+$targetConfig/}lib/lib*.la"  "''${!outputLib}"
+    # libssp_nonshared.a is needed because it is referenced by libssp.la
+    moveToOutput "''${targetConfig+$targetConfig/}lib/lib*.a"  "''${!outputLib}"
```

Comment:
Seeking feedback on this diff - I was unable to get various things on freebsd to build under the pure bootstrap without it, but I can't help but wonder if there's something obviously wrong about it.